crates/viewer/re_viewer/src/lib.rs

```diff
@@ -287,6 +287,18 @@ pub fn reset_viewer_persistence() -> anyhow::Result<()> {
         } else {
             re_log::info!("Rerun state was already cleared.");
         }
+
+        // Clear the default cache directory if it exists
+        //TODO(#8064): should clear the _actual_ cache directory, not the default one
+        if let Some(cache_dir) = re_viewer_context::AppOptions::default_cache_directory() {
+            if cache_dir.exists() {
+                if let Err(err) = std::fs::remove_dir_all(&cache_dir) {
```

Comment:
This is a race.

Better to always call `std::fs::remove_dir_all` and then ignore [io::ErrorKind::NotFound](https://doc.rust-lang.org/beta/std/io/enum.ErrorKind.html#variant.NotFound)